cuckoo/update.py

```diff
@@ -126,8 +127,8 @@ def many(
         )
 
     def many_distinct(
-        self: InnerUpdate,
-        updates: list[dict],
+        self,
+        updates: DISTINCT_UPDATES,
```

Comment:
sneaked in a typing update, since we actually use `many_distinct` in production now - when writing this, my assumption was that we would actually have no use for this method and was rather sloppy with it.